cogs/plugins.py

```diff
@@ -302,6 +302,14 @@ async def parse_user_input(self, ctx, plugin_name, check_version=False):
             plugin = Plugin(user, repo, plugin_name, branch)
 
         else:
+            if not self.bot.config.get("registry_plugins_only", False):
```

Comment:
Sorry for late comment.

Isn't this bit here supposed to be
```suggestion
            if self.bot.config.get("registry_plugins_only", False):
```
???

Because currently, I cannot add local plugins without setting anything for this config in `.env`.

Reply:
Yes, you're right, my mistake for hastily merging this PR. I have pushed a fix to master.